tests/restapi/conftest.py

```diff
@@ -0,0 +1,140 @@
+                        -subj '/CN=test.server.restapi.sonic' \
+                        -out restapiserver.csr"
+    localhost.shell(local_command)
+
+    # Sign server certificate
+    local_command = "openssl x509 \
+                        -req \
+                        -in restapiserver.csr \
+                        -CA restapiCA.pem \
+                        -CAkey restapiCA.key \
+                        -CAcreateserial \
+                        -out restapiserver.crt \
+                        -days 825 \
+                        -sha256"
+    localhost.shell(local_command)
+
+    # Create client key
+    local_command = "openssl genrsa -out restapiclient.key 2048"
+    localhost.shell(local_command)
+
+    # Create client CSR
+    local_command = "openssl req \
+                        -new \
+                        -key restapiclient.key \
+                        -subj '/CN=test.client.restapi.sonic' \
+                        -out restapiclient.csr"
+    localhost.shell(local_command)
+
+    # Sign client certificate
+    local_command = "openssl x509 \
+                        -req \
+                        -in restapiclient.csr \
+                        -CA restapiCA.pem \
+                        -CAkey restapiCA.key \
+                        -CAcreateserial \
+                        -out restapiclient.crt \
+                        -days 825 \
+                        -sha256"
+    localhost.shell(local_command)
+
+    # Copy CA certificate and server certificate over to the DUT
+    duthost.copy(src='restapiCA.pem', dest='/etc/sonic/credentials/')
+    duthost.copy(src='restapiserver.crt', dest='/etc/sonic/credentials/testrestapiserver.crt')
+    duthost.copy(src='restapiserver.key', dest='/etc/sonic/credentials/testrestapiserver.key')
+
+    # Set client certificate subject name in config DB
+    dut_command = "redis-cli -n 4 hset \
+                    'RESTAPI|certs' \
+                    'client_crt_cname' \
+                    'test.client.restapi.sonic'"
+    duthost.shell(dut_command)
+
+    # Set CA cert path in config DB
+    dut_command = "redis-cli -n 4 hset \
+                    'RESTAPI|certs' \
+                    'ca_crt' \
+                    '/etc/sonic/credentials/restapiCA.pem'"
+    duthost.shell(dut_command)
+
+    # Set server certificate path in config DB
+    dut_command = "redis-cli -n 4 hset \
+                    'RESTAPI|certs' \
+                    'server_crt' \
+                    '/etc/sonic/credentials/testrestapiserver.crt'"
+    duthost.shell(dut_command)
+    dut_command = "redis-cli -n 4 hset \
+                    'RESTAPI|certs' \
+                    'server_key' \
+                    '/etc/sonic/credentials/testrestapiserver.key'"
+    duthost.shell(dut_command)
+
+    # Restart RESTAPI server with the updated config
+    dut_command = "sudo systemctl restart restapi"
+    duthost.shell(dut_command)
+    time.sleep(RESTAPI_SERVER_START_WAIT_TIME)
```

Comment:
I cannot use wait_until here because, we cannot only look for restapi container to be running, but we need the process go-server-server to also be running which will take a few more seconds after the container starts.